mne_bids/write.py

```diff
@@ -289,6 +289,43 @@ def _events_tsv(events, durations, raw, fname, trial_type, overwrite=False):
     _write_tsv(fname, data, overwrite)
 
 
+def _events_json(fname, overwrite=False):
+    """Create participants.json for non-default columns in accompanying TSV.
+
+    Parameters
+    ----------
+    fname : str | mne_bids.BIDSPath
+        Output filename.
+    overwrite : bool
+        Whether to overwrite the output file if it exists.
+    """
+    new_data = {
+        'sample': {
+            'Description': 'The event onset time in number of sampling points.'
```

Comment:
needs clarification whether 0based or 1based indexing is used

Reply:
I don't know which one we use, and I don't know how it interacts with `first_samp`

Could you help?

Reply:
I don't know either, but let's merge this and make a new issue to improve on this later.  `raw.first_samp` is always confusing :)